ansible/README.md

```diff
@@ -0,0 +1,84 @@
+Deploying Openwhisk using Ansible
+=========
+
+
+### Getting started
+
+If you want to deploy Openwhisk locally using Ansible, you first need to install Ansible on your development environment:
+
+#### Ubuntu users
+```
+sudo apt-get install python-pip
+sudo pip install ansible==2.0.2.0
+```
+
+#### Vagrant users
+Nothing to be done, Ansible is already installed during vagrant provisioning.
+
+#### Mac users
+It is assumed that a VM has been provisioned using **docker-machine**.
+
+```
+brew install python
+pip install ansible==2.0.2.0
+
+cd ansible/environments/mac
+./genHosts.sh <name of docker-machine>
+```
+After this there should be a `hosts` file in the `ansible/environments/mac` dir.
+
+To verify the hosts file you can do a quick ping to the docker machine:
+```
+ansible all -i ansible/enviroments/mac -m ping
+```
+Should result in something like:
+```
+ansible | SUCCESS => {
+    "changed": false,
+    "ping": "pong"
+}
+192.168.99.100 | SUCCESS => {
+    "changed": false,
+    "ping": "pong"
+}
+```
+
+### Install Prerequisites
+This needs to be done only once per environment.
+```
+ansible-playbook -i ansible/environments/<environment> ansible/prereq.yml
+```
```

Comment:
it's a problem of ansible itself, ansible treats all executable scripts in environments folder as *dynamic inventory*

http://docs.ansible.com/ansible/intro_dynamic_inventory.html
```
In an inventory directory, executable files will be treated as dynamic inventory sources and most other files as static sources
```

Actually, we already set the flag `inventory_ignore_extensions` in `ansible.cfg` to tell ansible to ignore such files, it's also contained in your rebase?